zulipterminal/ui_tools/boxes.py

```diff
@@ -107,27 +107,30 @@ def autocomplete_mentions(self, text: str, state: int,
                               prefix_string: str) -> Optional[str]:
         # Handles user mentions (@ mentions and silent mentions)
         # and group mentions.
+        at_index = max(text.find(prefix_string), 0)
```

Comment:
A more generic name like `prefix_index` would be more descriptive here

Reply:
Thanks, `prefix_index` would be more appropriate.